src/api/resources/classification/image.py

```diff
@@ -0,0 +1,69 @@
+from django.db.models import Q
+from rest_condition import And
+from rest_framework import status
+from rest_framework.response import Response
+
+from ...exceptions import check_uuid
+from ...models import BENTHICPQT_PROTOCOL, CollectRecord, Image, ObsBenthicPhotoQuadrat
+from ...permissions import CollectRecordOwner
+from ...utils import truthy
+from ...utils.classification import classify_image
+from ..base import BaseAPISerializer, BaseProjectApiViewSet
+
+
+class ImageSerializer(BaseAPISerializer):
```

Comment:
So we'll handle potential nested serializers needed by frontend in a future ticket? We should reconvene with Parm and Julia; seems to me the CR should store an object that would make use of a list of annotations for each point in a list of points per image.